src/modules/export/latex.ts

```diff
@@ -21,12 +19,96 @@ export async function saveLatex(
     );
     await IOUtils.makeDirectory(attachmentsDir);
   }
-  await Zotero.File.putContentsAsync(
-    filename,
-    await addon.api.convert.note2latex(noteItem, dir, options),
+  const [latexContent, bibString] = await addon.api.convert.note2latex(
+    noteItem,
+    dir,
+    options,
   );
+  await Zotero.File.putContentsAsync(filename, latexContent);
 
   showHintWithLink(`Note Saved to ${filename}`, "Show in Folder", (ev) => {
     Zotero.File.reveal(filename);
   });
+
+  if (bibString && bibString.length > 0) {
+    const raw = await new ztoolkit.FilePicker(
+      `${Zotero.getString("fileInterface.export")} Bibtex File`,
+      "save",
+      [["Bibtex File(*.bib)", "*.bib"]],
+      `notegeneration.bib`,
+    ).open();
+    if (!raw) {
+      Zotero.debug("[Bib Export] Bib file export canceled.");
```

Comment:
Use `ztoolkit.log` whenever we can, which also outputs to the console in dev mode.